exonum-java-binding/testing/src/test/java/com/exonum/binding/test/runtime/ServiceArtifactBuilderTest.java

```diff
@@ -121,8 +121,8 @@ void createArtifactWithExtensionClass() throws IOException {
     assertThat(allJarEntries).containsKey(EXTENSIONS_INDEX_NAME);
     String allExtensions = getExtensionsIndex(allJarEntries);
 
-    String expectedExtensions = TestService.class.getName() + "\n"
-        + ImmutableList.class.getName() + "\n";
+    String expectedExtensions = TestService.class.getName() + System.lineSeparator()
```

Comment:
This guy uses the system line separator (and let it do that): https://github.com/exonum/exonum-java-binding/blob/404c816ac3436614626c7cedd473098dcd1e93b3/exonum-java-binding/testing/src/main/java/com/exonum/binding/test/runtime/ServiceArtifactBuilder.java#L242